spec/features/categories_show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.feature 'Categories', type: :system do
+  include Devise::Test::IntegrationHelpers
+  before do
+    @user = User.create(
+      name: 'Marwan',
+      email: 'user@example.com',
+      password: 'password',
+      confirmed_at: '2023-08-10 13:46:21.825849',
+      confirmation_sent_at: '2023-08-10 13:46:18.95477'
+    )
+    sign_in @user
+    @category =
+      Category.create(
+        name: 'cat 1',
+        icon: 'https://hips.hearstapps.com/hmg-prod/images/beautiful-smooth-haired-red-cat-lies-on-the-sofa-royalty-free-image-1678488026.jpg',
+        user: @user
+      )
+    @transaction =
+      Transaction.create(
+        name: 'tran 1',
+        amount: 15,
+        author: @user
+      )
+    @transaction_category =
+      TransactionCategory.create(
+        category: @category,
+        my_transaction: @transaction
+      )
+  end
+
+  describe '#show' do
+    it 'should display categories' do
+      visit category_path(@category)
+      expect(page).to have_content(@category.name)
+      expect(page).to have_css('img')
+      expect(page).to have_content(@category.transactions.sum(:amount))
+      expect(page).to have_content(@transaction.name)
+      expect(page).to have_content(@transaction.amount)
+      expect(page).to have_content('New Transaction')
+    end
```

Comment:
- [ ] Kindly add more test files and test models and other controllers.
- [ ] Kindly fix the tests which are not passing.

![test](https://github.com/MarwanDev/Tengo-Wallet/assets/53124548/8768d4be-9285-4318-ad7c-b19f6493b9e2)



![tests](https://github.com/MarwanDev/Tengo-Wallet/assets/53124548/e9c15469-02b4-4e29-b30f-e3122c9d8b2b)